charts/influxdb-enterprise/values.yaml

```diff
@@ -23,6 +23,14 @@ serviceAccount:
   name: ''
   annotations: {}
 
+## Customize liveness, readiness and startup probes
+## ref: https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/
+livenessProbe: {}
+  # path: "/ping"
+  # initialDelaySeconds: 30
+  # timeoutSeconds: 5
+  # scheme: HTTP
```

Comment:
Since this `scheme` isn't consumed I think this might result in an YAML error for the templating engine through `helm`.

I've not tried this myself, but at a quick glance seeing that `.Values.livenessProbe.scheme` isn't used in the `scheme` field might be problematic. I suspect the default can be `HTTP` in a similar manner to the `/ping` path

Reply:
Same with timeoutSeconds